packages/docusaurus-plugin-client-redirects/src/index.ts

```diff
@@ -46,6 +45,6 @@ export default function pluginClientRedirectsPages(
   };
 }
 
-export function trimBaseUrl(path: string, baseUrl: string): string {
-  return path.startsWith(baseUrl) ? path.replace(baseUrl, '/') : path;
+export function trimBaseUrls(paths: string[], baseUrl: string): string[] {
```

Comment:
??? :p

Reply:
I was hesitant to inline it because it looked messy 🤣 
I'll go and inline it now

Reply:
@teikjun actually I agree with you to not inline and write a test (using the leading / is not so obvious), it's just you didn't use the removePrefix method :p

Reply:
but that's not a big deal, I think we can merge as is

Reply:
Ohh, I see! I'll refactor it next time if I work on this part of the code again :)